lib/node-utils.ts

```diff
@@ -169,6 +212,49 @@ export function isPromiseResolved(node: TSESTree.Node): boolean {
   return hasThenProperty(parent);
 }
 
+/**
+ * Determines whether an Identifier related to a promise is considered as handled.
+ *
+ * It will be considered as handled if:
+ * - it belongs to `await` expression
+ * - it's chained with `then` method
+ * - it's returned from a function
+ * - has `resolves` or `rejects`
```

Comment:
```suggestion
 * It will be considered as handled if:
 * - it belongs to the `await` expression
 * - it's chained with the `then` method
 * - it's returned from a function
 * - has `resolves` or `rejects`
```